functions/Get-DbaQueryStoreConfig.ps1

```diff
@@ -0,0 +1,176 @@
+You should have received a copy of the GNU General Public License along with this program. If not, see http://www.gnu.org/licenses/.
+
+
+.LINK
+https://dbatools.io/Get-QueryStoreConfig
+
+.EXAMPLE
+Get-DbaQueryStoreConfig -SqlServer ServerA\sql
+
+Returns Query Store configuration settings for every Query Store enabled database on the ServerA\sql instance.
+
+	
+#>
+	[CmdletBinding()]
+	Param (
+		[parameter(Mandatory = $true, ValueFromPipeline = $true)]
+		[Alias("ServerInstance", "SqlInstance")]
+		[string[]]$SqlServer,
+		[PsCredential]$Credential
+	)
+
+	DynamicParam {
+		if ($SqlServer) {
+			return Get-ParamSqlDatabases -SqlServer $SqlServer[0] -SqlCredential $Credential
+		}
+	}
+
+	BEGIN
+	{
+		# Convert from RuntimeDefinedParameter object to regular array
+		$databases = $psboundparameters.Databases
+		$exclude = $psboundparameters.Exclude
+
+        # Create a datatable to hold the Query Store configuration
+        $QSCDT = New-Object System.Data.DataTable
+
+        # Define the columns
+        $QSCDT_Instance = New-Object System.Data.DataColumn �Instance�,([string])
+        $QSCDT_Database = New-Object System.Data.DataColumn �Database�,([string])
+        $QSCDT_ActualState = New-Object System.Data.DataColumn �ActualState�,([string])
+        $QSCDT_FlushInterval = New-Object System.Data.DataColumn �FlushInterval�,([string])
+        $QSCDT_CollectionInterval = New-Object System.Data.DataColumn �CollectionInterval�,([string])
+        $QSCDT_MaxSize = New-Object System.Data.DataColumn �MaxSize�,([string])
+        $QSCDT_CurrentSize = New-Object System.Data.DataColumn �CurrentSize�,([string])
+        $QSCDT_CaptureMode = New-Object System.Data.DataColumn �CaptureMode�,([string])
+        $QSCDT_CleanupMode = New-Object System.Data.DataColumn �CleanupMode�,([string])
+        $QSCDT_StaleQueryThreshold = New-Object System.Data.DataColumn �StaleQueryThreshold�,([string])
+
+        # Add columns to the datatable
+        $QSCDT.Columns.Add($QSCDT_Instance)
+        $QSCDT.Columns.Add($QSCDT_Database)
+        $QSCDT.Columns.Add($QSCDT_ActualState)
+        $QSCDT.Columns.Add($QSCDT_FlushInterval)
+        $QSCDT.Columns.Add($QSCDT_CollectionInterval)
+        $QSCDT.Columns.Add($QSCDT_MaxSize)
+        $QSCDT.Columns.Add($QSCDT_CurrentSize)
+        $QSCDT.Columns.Add($QSCDT_CaptureMode)
+        $QSCDT.Columns.Add($QSCDT_CleanupMode)
+        $QSCDT.Columns.Add($QSCDT_StaleQueryThreshold)
+	}
+
+	PROCESS
+	{
+		foreach ($servername in $SqlServer)
+		{
+            Write-Verbose "Connecting to $servername"
+			try
+			{
+				$server = Connect-SqlServer -SqlServer $servername -SqlCredential $Credential
+
+			}
+			catch
+			{
+				if ($SqlServer.count -eq 1)
+				{
+					throw $_
+				}
+				else
+				{
+					Write-Warning "Can't connect to $servername. Moving on."
+					Continue
+				}
+			}
+
+            $sqlVersion = $server.Version
+
+            $sqlVersion = $sqlVersion.ToString().Split(".")[0]
```

Comment:
You can exclude this line altogether by pulling the `VersionMajor` property instead of having to parse the output of `Version`:

![image](https://cloud.githubusercontent.com/assets/11204251/22994954/6bb7ed66-f38e-11e6-8596-fe6fbf2c8c34.png)